packages/storybook8/stories/Components/IncomingCallNotification/IncomingCallNotification/IncomingCallNotification.story.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IncomingCallNotification as IncomingCallNotificationComponent } from '@azure/communication-react';
+import React from 'react';
+
+const IncomingCallNotificationStory = (args: { callerName: string; showAcceptWithVideo: boolean }): JSX.Element => {
+  return (
+    <IncomingCallNotificationComponent
+      onAcceptWithAudio={function (): void {
+        alert('Accept with audio');
+      }}
+      onAcceptWithVideo={function (): void {
+        alert('Accept with video');
+      }}
+      onReject={function (): void {
+        alert('Rejected');
+      }}
+      callerName={args.callerName}
+      acceptOptions={{
+        showAcceptWithVideo: args.showAcceptWithVideo
+      }}
+    />
+  );
+};
+
+export const IncomingCallNotification = IncomingCallNotificationStory.bind({});
```

Comment:
Why do we bind {} is it a storybook specific syntax?

Reply:
Yeah all of the stories seem to have always done this, I think there are some options you can pass there but it can't by undefined?

Reply:
Just checked some docs:
```
Template.bind({}) is a standard JavaScript technique for making a copy of a function. We use this technique to allow each exported story to set its own properties, but use the same implementation.
```
https://storybook.js.org/tutorials/intro-to-storybook/ember/en/simple-component/